lib/po.js

```diff
@@ -234,7 +236,24 @@ PO.parse = function (data) {
     return po;
 };
 
-PO.Item = function () {
+PO.parsePluralForms = function (pluralFormsString) {
+    var results = (pluralFormsString || '')
+        .split(';')
+        .reduce(function (acc, keyValueString) {
+            var trimmedString = keyValueString.trim();
+            var equalsIndex = trimmedString.indexOf('=');
+            var key = trimmedString.substring(0, equalsIndex).trim();
+            var value = trimmedString.substring(equalsIndex + 1).trim();
+            acc[key] = value;
+            return acc;
+        }, {});
+    return {
+        nplurals: results.nplurals,
+        plural: results.plural
+    };
+};
+
+PO.Item = function (nplurals) {
```

Comment:
Indeed. With “strange” I probably meant “verbose”. If you don't mind changing it, I'd prefer the “object” variant.